src/components/TransactionsInterpreter/helpers/getTransactionTokens.ts

```diff
@@ -0,0 +1,22 @@
+import { UITransactionType } from './types';
+
+export const getTransactionTokens = (transaction: UITransactionType) => {
+  if (transaction.action) {
+    return [
+      ...(transaction.action.arguments?.token
```

Comment:
There has to be a cleaner way for this. maybe add them all to an array and filter by non-null?